frontend/model/translation.lem

```diff
@@ -359,7 +359,7 @@ let translate_div_mod_operator loc translate_expr usual_arithmetic_conversion st
     (usual_arithmetic_conversion (ctype_of e1) (ctype_of e2) obj1_wrp.E.sym_pe obj2_wrp.E.sym_pe) in
   let (ub, core_pe) = match aop with
     | A.Div ->
-        ( Undefined.UB045a_division_by_zero
+        ( Undefined.UB045a_division_by_non_positive
```

Comment:
4/-2 is allowed as is 4/-2. Please revert and fix check.

Reply:
and all the changes to `UB045a_division_by_zero` in other files and the change to the text of ISO standard in `tools/n1570.json`